src/Http/Routing/src/Builder/FallbackEndpointRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// Contains extension methods for <see cref="IEndpointRouteBuilder"/>.
+    /// </summary>
+    public static class FallbackEndpointRouteBuilderExtensions
+    {
+        /// <summary>
+        /// Adds a specialized <see cref="RouteEndpoint"/> to the <see cref="IEndpointRouteBuilder"/> that will match
+        /// requests for non-filenames with a very low priority.
```

Comment:
`a very low priority` sounds a bit imprecise. Since it's `Order = int.MaxValue`, how about `the lowest possible priority`?

Reply:
Lower number > priority. 0 > 1 if that makes sense. It's been how routing has worked since forever

Reply:
All I'm commenting on here is the phrasing used in the XML docs. `a very low priority` is an imprecise statement (just *how* low is the priority?), whereas `the lowest possible priority` tells you something exact.

Reply:
sure